components/Comments/components/Form/Form.tsx

```diff
@@ -40,6 +42,13 @@ export const Form: FC<IFormProps> = () => {
     dispatch(closeCommentsSidebar());
   };
 
+  useEffect(() => {
+    const listener = addKeyboardListener({ key: 'Enter', ctrl: true }, () =>
+      submitHandler({ text: watch('text') }),
+    );
+    return listener.clear();
```

Comment:
The same changes from https://github.com/Juratbek/udas_frontend/pull/596 should be applied for this PR